src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -666,7 +686,7 @@ class ValidatorStore extends BaseStore {
         }
         let keysharePayload;
         try {
-          keysharePayload = await keyShares.buildPayload({
+          keysharePayload = await (new KeySharesItem()).buildPayload({
```

Comment:
Use two lines, first with class instantiation instead of this less readable construction